ui/tests/integration/components/pki/pki-issuer-cross-sign-test.js

```diff
@@ -329,10 +329,10 @@ module('Integration | Component | pki issuer cross sign', function (hooks) {
       .dom(`${SELECTORS.signedIssuerRow()} [data-test-icon="alert-circle-fill"]`)
       .exists('row has failure icon');
     assert
-      .dom('[data-test-alert-banner="alert"] .message-title')
+      .dom('[data-test-alert-banner] .message-title')
```

Comment:
I'm glad you brought this up! I noticed in the past that general selectors tended to be harder to leverage in tests because they'd appear a bunch of times. Especially with components like alerts where multiple may appear on a page. I'm hoping to actually remove the general `data-test-alert-banner` all together and have each component reference a unique selector. Ideally this also makes debugging test easier!